AuxTel/AuxTel-Non-standard-Operations/AuxTel-EUI-Access/AuxTel-EUI-Access.rst

```diff
@@ -0,0 +1,107 @@
+.. This is a template for an informative/general use document. 
+
+.. Review the README in this document's directory on instructions to contribute.
+.. Static objects, such as figures, should be stored in the _static directory. Review the _static/README in this procedure's directory on instructions to contribute.
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: Kshitija Kelkar
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *List-of-contributors*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _Templates-Informative-Document:
```

Comment:
Add label for future reference; suggested, AuxTel-Non-Standard-Operations-AuxTel-EUI-Access: